dGame/LeaderboardManager.cpp

```diff
@@ -150,14 +156,44 @@ Leaderboard *LeaderboardManager::GetLeaderboard(uint32_t gameID, InfoType infoTy
 
     std::string query;
     switch (infoType) {
+
+        // Modified 12-12-2021: Not all gamemodes are dictated by the score. Instead, the time dictates the best score.
         case InfoType::Standings:
-            query = leaderboardType == MonumentRace ? standingsQueryAsc : standingsQuery;
+            switch (leaderboardType) {
+                case ShootingGallery: 
+                    query = standingsScoreQuery; // Shooting gallery is based on the highest score.
+                    break;
+                case Survival:
+                    query = standingsTimeQuery; // AG Survival is based on the longest time.
```

Comment:
Again, same issue. While it started as time only, they changed it late into LU's service to be time + score.